tests/integration-tests/conversations/test_concurrent_stream_timing.py

```
@@ -141,31 +151,27 @@ async def test_concurrent_stream_timing_before_vs_after(
     url = f"/api/v1/conversations/{conversation_id}/message"
     form_data = {"content": "Timing before/after test."}
 
-    # Simulate "before" fix: make to_thread run the callable on the event loop (blocking)
-    async def _blocking_impl(f, *args, **kwargs):
-        return f(*args, **kwargs)
-
-    def blocking_to_thread(f, *args, **kwargs):
-        return _blocking_impl(f, *args, **kwargs)
+    async def blocking_wait_for_task_start(*args, **kwargs):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `blocking_wait_for_task_start`

Add return type annotation: `bool`

(ANN202)